### DISCUSSION TITLE
PropertyDecorators such as @AfterInsert()

### DISCUSSION THREAD

**Discussion by ghost** [MichalLytek/typegraphql-prisma#296] [Q&A] [answered]
Hello,

I am currently switching out TypeORM for Prisma and was wondering if PropertyDecorators such as `@AfterInsert()` are supported?

As always, thank you!

**Comment by MichalLytek** [comment, marked as answer]
This is only a GraphQL API, not an ORM adapter for Prisma. While you can add some middlewares for resolvers, it won't work when you call prisma client in services.

**Reply by ghost** [reply]
Great, thank you!
